clock_exercise/ActivationAlarmSwitch.qml

```diff
@@ -0,0 +1,55 @@
+import QtQuick 2.0
+
+Rectangle {
+    id: root
+
+    property bool on: false
+
+    signal changed
+
+    width: 112
+    height: 60
+    radius: 8
+    color: "#339FAAB0"
+
+    Image {
+        x: 70
+        y: 17
+        width: 26
+        height: 26
+        source: "images/223-bell.svg"
+        opacity: 0.3
+    }
+
+    Rectangle {
+        x: 30
+        y: 18
+        width: 2
+        height: 22
+        color: "black"
+        opacity: 0.25
+    }
+
+    Image {
+        x: 4
+        y: 4
+        width: 52
+        height: 52
+        source: "images/switch-off-selected.svg"
+        visible: !root.on
+    }
+
+    Image {
+        x: 56
+        y: 4
+        width: 52
+        height: 52
```

Comment:
In generale evitiamo di specificare le dimensioni se un asset è già delle dimensioni giuste come in questo caso: aprendo le proprietà del file su disco viene infatti riportato 52x52.
Solitamente sta ai designer esportare gli asset nelle dimensioni giuste, quindi posso supporre che se devo usare una particolare immagine, quella sia già delle dimensioni che mi sono state richieste dai documenti di design (tipo le schermate su Adobe XD).